dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/program/notification/ProgramNotificationTemplateOperationParamsMapper.java

```diff
@@ -58,8 +58,8 @@ public ProgramNotificationTemplateQueryParams map(
         .programStage(programStage)
         .page(operationParams.getPage())
         .pageSize(operationParams.getPageSize())
-        .paged(operationParams.isPaged())
-        .skipPaging(operationParams.isPaged())
+        .paged(operationParams.isPaging())
+        .skipPaging(operationParams.isPaging())
```

Comment:
Shouldn't be the opposite here?

```suggestion
        .skipPaging(!operationParams.isPaging())
```

Reply:
Well spotted! I'll remove the field as its unused.